openstates/importers/base.py

```diff
@@ -328,7 +328,8 @@ def import_item(self, data: _JsonDict) -> typing.Tuple[_ID, str]:
         data = self.apply_transformers(data)
         try:
             data = self.prepare_for_db(data)
-        except UnresolvedIdError:
```

Comment:
Should we catch _all_ errors here? Perhaps the stacktrace was intentional? I know in a lot of cases James designed the systems to fail on exception.